Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php

```diff
@@ -272,4 +272,35 @@ public function testWrongCasedPhpdocTagIsNotAltered()
 EOF;
         $this->makeTest($expected);
     }
+
+    public function testInlineDoc()
+    {
+        $expected = <<<'EOF'
+<?php
+    /**
+     * Does stuffs with stuffs.
+     *
+     * @param array $stuffs {
+     *     @type bool $foo
+     *     @type int $bar
```

Comment:
Right. Can we leave that to another PR, since I have nothing to do with the alignment fixer. TBH, I'd still like to see that rewritten to use the phpdoc classes we have for consistent behaviour.